### PR TITLE
Removed phone, Updated README.md and updated String.xml with new info

### PR DESCRIPTION
**Pull request by KartikShankhavaram**
Updated names of gymkhana heads, their photos, emails. Also updated Names and emails of Administration. Removed all instances of contact numbers from strings.xml and the code. Corrected syntax errors in README.md and modified mentors. Issue #5 not currently resolved.

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -12,12 +12,7 @@
             <option value="$PROJECT_DIR$/app" />
           </set>
         </option>
-        <option name="myModules">
-          <set>
-            <option value="$PROJECT_DIR$" />
-            <option value="$PROJECT_DIR$/app" />
-          </set>
-        </option>
+        <option name="resolveModulePerSourceSet" value="false" />
```
Comment (saketkumar95) [inline .idea/gradle.xml]: Why this changes? Could you please explain?

File: .idea/modules.xml [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/LNMIIT_Android_App.iml" filepath="$PROJECT_DIR$/LNMIIT_Android_App.iml" />
+      <module fileurl="file://$PROJECT_DIR$/LNMIIT_Android_App.iml" filepath="$PROJECT_DIR$/LNMIIT_Android_App.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```
Comment (saketkumar95) [inline .idea/modules.xml]: Remove this auto generated changes from this PR.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -19,6 +19,7 @@
         <activity
             android:name=".activity.MainActivity"
             android:label="@string/app_name"
+            android:screenOrientation="portrait"
```
Comment (saketkumar95) [inline app/src/main/AndroidManifest.xml]: Why bounding the app only to portrait?

Comment (KartikShankhavaram) [reply app/src/main/AndroidManifest.xml]: A temporary fix while I figure out how to fix this issue

File: .idea/gradle.xml (hunk 1 above)
Comment (KartikShankhavaram) [reply .idea/gradle.xml]: Beacuse of the new .gitignore, done automatically

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (saketkumar95) [reply app/src/main/AndroidManifest.xml]: Okay got that! Let's go take this in. 👍

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -89,23 +89,6 @@
         <item>jadonmukesh30@gmail.com</item>
     </string-array>
 
-    <string-array name="cse_faculty_contact">
```
Comment (saketkumar95) [inline app/src/main/res/values/strings.xml]: I'm not checking the data currently. I hope you checked everything is correct.

Comment (KartikShankhavaram) [reply app/src/main/res/values/strings.xml]: I updated all the data that I know is correct, haven't updated the data about the faculties.

File: .idea/modules.xml (hunk 2 above)
Comment (KartikShankhavaram) [reply .idea/modules.xml]: removed

**Comment by saketkumar95** [comment]
Merged! 👍
